content/en/platform/corda/5.0/developing-applications/api/api-crypto.md

```diff
@@ -8,15 +8,21 @@ menu:
     parent: corda5-api
     weight: 5000
 section_menu: corda5
-draft: true
 ---
 # net.corda.v5.crypto
 
-The `corda-crypto` module 
+The `crypto` package provides services and types for performing cryptographic operations. The following services are available:
+
+* `CompositeKey` is a tree data structure that enables the representation of composite public keys, which are used to represent the signing requirements for multi-signature scenarios.
```

Comment:
I understand this is as mentioned on the kDoc but sounds rather an implementation detail. Maybe let's leave as is for now (to be aligned with the kDoc as well) but maybe we should update the kDoc and this to begin with something more generic/ simple of what a `CompositeKey` is.